bayesian_cnn_prometheus/analysis/analyze_masks.py

```diff
@@ -0,0 +1,13 @@
+from bayesian_cnn_prometheus.analysis.correlation_analyzer import CorrelationAnalyzer
+
+
+def main():
+    variance_masks_path = "/home/szymon/Pulpit/AILab/3d-cnn-prometheus/bayesian_cnn_prometheus/data/SEGMENTATION_VARIANCE_0003.nii.gz"
+    cancer_masks_path = "/home/szymon/Pulpit/AILab/3d-cnn-prometheus/bayesian_cnn_prometheus/data/IMG_0003.nii.gz"
```

Comment:
Tu może podmień na jakieś deskryptywne stringi :D Wiem, że to nie jest końcowe rozwiązanie (pasowałoby to z CLI odpalać i to pewnie na folderach z wariancjami i obrazkami, a nie na pojedynczych),  ale tak tymczasowo może być przyjemniej :)